src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```
@@ -553,8 +553,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ItemGroup>
       <_ResolvedCopyLocalPublishAssets Include="@(ReferenceCopyLocalPaths)"
                                        Exclude="@(_ResolvedCopyLocalBuildAssets);@(RuntimePackAsset)"
-                                       Condition="'$(PublishReferencesDocumentationFiles)' == 'true' or '%(Extension)' != '.xml'">
-        <DestinationSubPath>%(ReferenceCopyLocalPaths.DestinationSubDirectory)%(Filename)%(Extension)</DestinationSubPath>
+                                       Condition="'$(PublishReferencesDocumentationFiles)' == 'true' or '%(ReferenceCopyLocalPaths.Extension)' != '.xml'">
+        <DestinationSubPath>%(ReferenceCopyLocalPaths.DestinationSubDirectory)%ReferenceCopyLocalPaths.Filename)%(ReferenceCopyLocalPaths.Extension)</DestinationSubPath>
```

Is this not missing a `(`?

```suggestion
        <DestinationSubPath>%(ReferenceCopyLocalPaths.DestinationSubDirectory)%(ReferenceCopyLocalPaths.Filename)%(ReferenceCopyLocalPaths.Extension)</DestinationSubPath>
```

---

No idea how the test passed. Will investigate. Correction applied.

---

It looks like `DestinationSubPath` isn't used, and `RelativePath` is set based on `DestinationSubDirectory` [here](https://github.com/dotnet/sdk/blob/c561c3f7e1ad8da7db424c75338c9d81623b2d99/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L410).

---

That is to say, it isn't being used by the targets that follow.  I think it would get used for deps file generation, but that doesn't depend on this target.